cmd/stolon-pgbouncer/main.go

```diff
@@ -489,6 +529,22 @@ func main() {
 	logger.Log("event", "shutdown")
 }
 
+// Renders a HealthCheckResponse as human-readable text to stdout
```

Comment:
Is it worth having a package function in `failover` that can generate a string, then print the result of that here? Would mean we can easily add a quick unit test for presenting the status without dealing with IO, and it feels something like this could benefit from a unit test to demonstrate its structure.